core-backend/sandbox/src/env.rs

```diff
@@ -161,6 +161,12 @@ where
             Funcs::reservation_reply_commit,
         );
         builder.add_func("gr_reply_push", Funcs::reply_push);
+        builder.add_func("gr_rereply", Funcs::rereply);
```

Comment:
i think naming should be made more consistent

for example,`gr_reply_push` should have an analog of `gr_reply_push_input` and so on

Reply:
Yep, agree. What is about 'gr_relay_{send, push}`?

Reply:
Relay has too opionated meaning, it's not just about relay